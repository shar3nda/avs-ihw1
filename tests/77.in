10
1 10 4 0 5 10 7 3 2 8