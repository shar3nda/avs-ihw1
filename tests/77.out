-2 4 -3 0 -7 -2 1 -4 9 9